app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEdit.scala.html

```diff
@@ -114,7 +114,7 @@
             title = Text("There is a problem")
         ))
     }
-    <h1 class="govuk-heading-l">@heading</h1>
+    <h1 class="govuk-heading-l">@messages("edit-set.record.edit.heading", editSetRecordForm.data.get("ccr").get)</h1>
```

Comment:
`.get` on option is being used elsewhere, maybe a sensible default / cleaner way to fail would be a good idea for a future ticket?

Reply:
Could we merely default to `""`?